mPower2/mPower2/View Controllers/StudyBurstCompletionViewController.swift

```diff
@@ -0,0 +1,102 @@
+//
+//  StudyBurstCompletionViewController.swift
+//  mPower2
+//
+//  Created by Josh Bruhin on 6/7/18.
+//  Copyright © 2018 Sage Bionetworks. All rights reserved.
+//
+
+import UIKit
+import BridgeApp
+
+class StudyBurstCompletionViewController: UIViewController {
+
+    
+    @IBOutlet weak var titleLabel: UILabel!
+    @IBOutlet weak var textLabel: UILabel!
+    @IBOutlet weak var detailLabel: UILabel!
+    @IBOutlet weak var navFooterView: RSDGenericNavigationFooterView!
+    @IBOutlet weak var backButton: UIButton!
+    @IBOutlet weak var statusBarBackgroundView: RSDStatusBarBackgroundView!
+    
+    var surveyManager: SurveyScheduleManager?
+
+    static func instantiate() -> StudyBurstCompletionViewController? {
+        let storyboard = UIStoryboard(name: "Main", bundle: nil)
+        return storyboard.instantiateViewController(withIdentifier: "StudyBurstCompletionViewController") as? StudyBurstCompletionViewController
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        // Do any additional setup after loading the view.
+        setupView()
+    }
+
+    func setupView() {
+        
+        // Populate the labels
+        titleLabel.text = Localization.localizedString("STUDY_BURST_COMPLETION_TITLE")
+        textLabel.text = Localization.localizedString("STUDY_BURST_COMPLETION_TEXT")
+        detailLabel.text = Localization.localizedString("STUDY_BURST_COMPLETION_DETAIL")
+
+        // Configure the next button
+        navFooterView.nextButton?.addTarget(self, action: #selector(nextHit(sender:)), for: .touchUpInside)
+        
+        // Show or hide the next button
+        navFooterView.isHidden = !hasActiveSurvey
+        
+        // Assign color to the status bar background view
+        statusBarBackgroundView.overlayColor = UIColor.rsd_statusBarOverlayLightStyle
+    }
+    
+    var hasActiveSurvey : Bool {
+        return surveyManager?.hasSurvey ?? false
+    }
+
+    
+    // MARK: Actions
+    @objc
+    func nextHit(sender: Any) {
+        
+        guard let surveyManager = surveyManager,
+            let group = surveyManager.activityGroup,
+            let taskInfo = group.tasks.first else {
+            return
+        }
```

Comment:
If they tap the next button then **something** should happen, yeah? dismiss the view controller?